### ISSUE TITLE
More Configuration Ability

### ISSUE THREAD

**Issue by modernham** [ampledata/aprscot#14]
Any way we can get some more of the configuration options moved to the ini?
We are missing really basic configuration options like connection methods built into PyTak such as TLS, and the ability to change call signs/ APRIS hosts that tare all hard-coded into the library itself. 
It's kind of a deal breaker for the library right now for me.

**Comment by Zuilf456WQ** [comment]
Yup, the config file do not match what the python program is waiting for parameters. 

this is a compatible config file :

[aprscot]

# APRSCOT Configuration: https://aprscot.rtfd.io/
# Change values here, and restart with: sudo systemctl restart aprscot

# Enable or Disable the APRSCOT software daemon.
ENABLED=1

# URL to CoT destination. Must be a URL, e.g. tcp://1.2.3.4:1234 or tls://...:1234, etc.
COT_URL=tls://yourtakserver.com:8089

# APRS-IS Login Callsign
CALLSIGN=YOURCALLSIGN (dosen't matter for recieve only)

# APRS-IS Passcode ("password")
APRSIS_PASSCODE=-1

# APRS-IS Server-side Filter. See: http://www.aprs-is.net/javAPRSFilter.aspx
APRSIS_FILTER = r/XX.XX/YY.YY/50 m/1000

# APRS-IS Server
APRS_HOST=aprs.glidernet.org

# APRS-IS Port
APRS_PORT=14580

MAX_IN_QUEUE=20000
MAX_OUT_QUEUE=2000

# COT Stale period ("timeout"), in seconds.
COT_STALE=20

# Override COT Event Type ("marker type").
COT_TYPE=a-f-G-I-U-T-r


# PyTAK Configuration: https://pytak.rtfd.io/
# Change values here, and restart with: sudo systemctl restart aprscot

# Sets TAK Protocol to use for CoT output.
TAK_PROTO=0

# Sets debug-level logging. Any value other than 0 is considered True. False if unset.
DEBUG=1

# TAK Data Packages containing TAK Server connection settings, TLS certificates, etc.
# PREF_PACKAGE=

# Path to a file containing the unencrypted plain-text PEM format Client Certificate.
# This file can contain both the Client Cert & Client Key, or the Client Cert alone.
# In the later case (cert alone), PYTAK_TLS_CLIENT_KEY must be set to the Client Key.
PYTAK_TLS_CLIENT_CERT= yourcert.pem

# Path to a file containing the unencrypted plain-text PEM format Client Private Key
# for the associated PYTAK_TLS_CLIENT_CERT.
PYTAK_TLS_CLIENT_KEY= yourkey.key

# Password for password protected certificates or password protected Private Keys.
# PYTAK_TLS_CLIENT_PASSWORD=

# Disable destination TLS Certificate Verification.
PYTAK_TLS_DONT_VERIFY=1

# Disable destination TLS Certificate Common Name (CN) Verification.
PYTAK_TLS_DONT_CHECK_HOSTNAME=1

# File containing the CA Trust Store to use for remote certificate verification.
# PYTAK_TLS_CLIENT_CAFILE=

# Expected hostname or CN of the connected server. Not used unless verifying hostname.
# PYTAK_TLS_SERVER_EXPECTED_HOSTNAME=





But, it seems that there is a problem woth the queue taht is goig to full verry quickly. Try to debug it.
